scripts/update-bootnodes.sh

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env bash
+#
+# Update ETC bootnodes in the configuration file
+# This script fetches active bootnodes from the etcnodes API
+# and updates the configuration to maintain 20 active bootnodes at all times.
```

Comment:
we should add 1.5x the number of boot nodes as the default max outgoing connection is